system-addon/content-src/components/Base/Base.js

```diff
@@ -0,0 +1,3 @@
+const React = require("react");
+const Base = () => <div>New Tab</div>;
```

Comment:
This is supposed to have brackets around it according to our lint settings... I think. You might want to wait until ed's lint patch lands just to make sure